facebook/micro/monthly-archive.sh

```diff
@@ -26,5 +26,5 @@ perform_rollup_and_post ()
     mv *.gz $1/
 }
 
-perform_rollup_and_post "fb-public-results"
-perform_rollup_and_post "protected-race-ethnicity-data"
+perform_rollup_and_post "fb-public-results" ""
```

Comment:
the "" is not technically necessary but I'm okay with keeping it in for extra readability